### DISCUSSION TITLE
Secure presenter mode with password on built (production ready) presentation

### DISCUSSION THREAD

**Discussion by nejci** [slidevjs/slidev#889] [Q&A]
First, thank you @antfu and others for beautiful piece of software you built!

I would like to use **presenter mode** when presenting and I would like to **share my slides with the audience** so they can access them on their phone/tablet/laptop. I would like access to presenter mode to be **password protected**. 

As far as I know, there are two options:
1. Running the presentation in dev mode and pass the `--remote=password` parameter, e.g. `npx slidev --remote=password slides.md`. The audience can access the presentation, but it is in dev mode and **allows editing with the integrated CodeMirror editor**! It's not a good idea for my students to modify the content of my presentation during lectures 😉 

2. Building the presentation with `npx slidev build` and either upload it to a public server or serve it on a local network (e.g. using `serve`). In this case the presenter mode is not password protected and anyone in the audience can manage the presentation, right?

Is there a third option that would solve my problem? Am I missing something here?

Thank you in advance!

**Comment by guan4tou2** [comment]
i have a solution for this problem, but it's not office solution.  
open file 'node_modules/@slidev/client/internals/Editor.vue', then find line 61 'content.value = v' and comment out.

**Reply by nejci** [reply]
Thank you very much @guan4tou2 !
